src/styles/Blocks/stars.scss

```diff
@@ -0,0 +1,40 @@
+.product {
+
+  &__stars-reviews {
+    display: flex;
+    margin-bottom: 24px;
+    flex-direction: row;
+    justify-content: space-between;
+
+      .stars {
```

Comment:
`stars` is a separate BEM block, move it from `product`. And why the file is called stars, but BEM block is product 🤷‍♂️